config/config.form.yaml

```
@@ -180,6 +180,8 @@ sector:
   reduce_space_heat_exogenously_factor: # TODO: check this for Germany (Regulation or Ariadne?)
     2035: 0.11
   tes: false
+  dac: true
+  dac_limit: false  # False or limit in MtCO2
```

atm, this is specified as either false or the limit in MtCO2. An alternative could be to include multiple levels such as `enable` and then optimization years.
```suggestion
  dac_limit:
      enable: False
      2035: 20  # limit in MtCO2
```

---

I like this one !